content/razor-views/reading/iterating-templates/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "Iterating in a Template"
+date: 2023-02-09T12:48:24-06:00
+draft: false
+weight: 4
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Let’s revisit part of the non-efficient HTML from the introduction, where we hard-coded coffee options in a list.
+
+```csharp{linenos=table,hl_lines=[],linenostart=1}
+<ol>
+   <li>French Roast</li>
+   <li>Espresso</li>
+   <li>Kopi Luwak</li>
+   <li>Instant</li>
+</ol>
+```
+
```

Comment:
Leave it up to you, but think blue example boxes might make it easier to follow by breaking up the concepts and code a bit more.